### ISSUE TITLE
problem with get_azimuth_fast

### ISSUE THREAD

**Issue by fsteinmetz** [pingswept/pysolar#75]
The values returned by `get_azimuth_fast` are completely different from those returned by `get_azimuth`. After verification it seems that `get_azimuth_fast` is the culprit. Offset of 180 ?

```python
from pysolar import solar
import numpy as np
from datetime import datetime
import pytz

lat = 50.
lon = 3.
time = datetime(2018, 5, 8, 12, 0, 0, tzinfo=pytz.UTC)

print(solar.get_azimuth(lat, lon, time))        # returns 186.8391776928919
print(solar.get_azimuth_fast(lat, lon, time))   # returns 6.8545786939043`
```

**Comment by pingswept** [comment]
Yes, I bet you have diagnosed the problem correctly. I'm not sure exactly where to fix the error. This line looks suspicious: https://github.com/pingswept/pysolar/blob/master/pysolar/solar.py#L174

**Comment by AmitAronovitch** [comment]
added fix for this issue into the PR of #81
